ivy_tests/test_docstrings.py

```diff
@@ -60,22 +60,16 @@ def test_docstrings(backend):
 
     # skip list for array and container docstrings
     skip_arr_cont = [
-        "cross_entropy",
```

Comment:
Please revert all changes made to this file as it's not related to the function you're working on